frappe/desk/form/document_follow.py

```diff
@@ -236,6 +240,8 @@ def get_added_row(added, time, doctype, doc_name, v):
 	return items
 
 def get_field_changed(changed, time, doctype, doc_name, v):
+	from html2text import html2text
```

Comment:
The same package is imported in multiple functions. How about adding lazy imports next to module level imports. something like

```
html2text = lazy_import('html2text') 
```
Here `html2text` will imported when an attribute from that module is called for the first time. and as it is at top level it is accessible to all the objects in that module.